gemp-lotr/gemp-lotr-async/src/main/web/src/updated/jCard/modules/image-resolution/data/hobbit.list.ts

```diff
@@ -0,0 +1,1023 @@
+import { Override } from "../types/override.interface";
```

Comment:
these should basically all be JSON files, but by the time I realized that I was just generating them as JSON arrays.  It ends up working out well in the end (image-resolver.functions.ts)

Reply:
oh, ignore. Unless you want to spot check